DOCS-910: Updating proxy and console configurations (#921)
minio/docs: source/reference/minio-server/minio-server.rst

.. envvar:: MINIO_CONFIG_ENV_FILE

   Specifies the full path to the file the MinIO server process uses for loading environment variables.
   

>    For ``systemd``-managed file, setting this value to the environment file allows MinIO to reload changes to that file on using :mc-cmd:`mc admin service restart` to restart the deployment.

Reviewer: ```suggestion
   For ``systemd``-managed files, setting this value to the environment file allows MinIO to reload changes to that file on using :mc-cmd:`mc admin service restart` to restart the deployment.
```

Not sure if this might be `For systemd-managed files` or `For a systemd-managed file`, but here's a suggestion for the first one.

.. envvar:: MINIO_ILM_EXPIRY_WORKERS

   Specifies the number of workers to make available to expire objects configured with ILM rules for expiration.
   When not set, MinIO defaults to using up to half of the available processing cores available.

.. envvar:: MINIO_DOMAIN

   Set to the Fully Qualified Domain Name (FQDN) MinIO accepts Bucket DNS (Virtual Host)-style requests on.

>    For example, setting ``MINIO_DOMAIN=minio.example.net`` would direct MinIO to accept an incoming connection request the ``data`` bucket at ``data.minio.example.net``.

Reviewer: ```suggestion
   For example, setting ``MINIO_DOMAIN=minio.example.net`` directs MinIO to accept an incoming connection request the ``data`` bucket at ``data.minio.example.net``.
```

>    The default behavior with this setting omitted is to accept only path-style requests as ``minio.example.net/data``.

Reviewer: Maybe? 
```suggestion
   If this setting is omitted, the default is to only accept path-style requests. For example, ``minio.example.net/data``.
```
